### ISSUE TITLE
Software is just data?

### ISSUE THREAD

**Issue by LourensVeen** [esciencecenter-digital-skills/research-software-support#60]
Regarding https://github.com/esciencecenter-digital-skills/research-software-support/blob/main/modules/researchsoftware/slides-software.pmd, I'm wondering where you found a computer scientist that said that software is just data :smile:. There are whole branches of computer science dedicated to algorithms and software engineering and programming languages and such to which software is not just some random ones and zeros.

If anything, it's maybe data stewards and librarians that see software as just another work to be catalogued and archived securely, and since they're the primary audience of this course, they do need to hear this, but I'm not sure how the computer scientist got involved :smile:.

**Comment by JaroCamphuijsen** [comment]
Yeah I already got corrected by Rob. I'll change it.

**Comment by JaroCamphuijsen** [comment]
indeed, this has been fixed long ago and but was never linked to the PR. i'll close it now.
